metadoc-js/src/main/scala/monaco/Monaco.scala

```diff
@@ -11,12 +11,12 @@ import monaco.languages.Languages.Definition
 import monaco.editor.Editor.BuiltinTheme
 import monaco.editor.Editor.LineNumbersOption
 
-@js.native
+@ScalaJSDefined
```

Comment:
Is this needed? If we can get away with only using monaco.Promise I think that is preferable.